react/Button/Button.js

```diff
@@ -10,9 +10,13 @@ export default class Button extends Component {
   static displayName = 'Button';
 
   static propTypes = {
-    color: PropTypes.oneOf([
-      'pink', 'blue', 'gray', 'transparent'
-    ]).isRequired,
+    /* eslint-disable consistent-return */
+    color: (props, propName, componentName) => {
+      if (!props.group && ['pink', 'blue', 'gray', 'transparent'].indexOf(props.color) === -1) {
```

Comment:
`!props.group &&`

Maybe this is an rfc concern and I have no issue with the above implementation just to be clear, but I could imagine a world where the following is quite useful:

```js
<Button group color="pink">
  <Button />
  <Button />
  <Button />
</Button>
```